webview-ui/src/components/chat/BatchFilePermission.tsx

```diff
@@ -35,10 +36,13 @@ export const BatchFilePermission = memo(({ files = [], onPermissionResponse, ts
 								<ToolUseBlockHeader
 									onClick={() => vscode.postMessage({ type: "openFile", text: file.content })}>
 									{file.path?.startsWith(".") && <span>.</span>}
-									<span className="whitespace-nowrap overflow-hidden text-ellipsis text-left mr-2 rtl">
-										{removeLeadingNonAlphanumeric(file.path ?? "") + "\u200E"}
-										{file.lineSnippet && ` ${file.lineSnippet}`}
-									</span>
+									<StandardTooltip
+										content={`${file.path}${file.lineSnippet ? ` ${file.lineSnippet}` : ""}`}>
```

Comment:
[P2] Same issue here. Guard against undefined and add an LRM so the tooltip mirrors the RTL-safe rendering used for the visible text.

```suggestion
<StandardTooltip content={`${(file.path ?? "")}${file.lineSnippet ? ` ${file.lineSnippet}` : ""}${(file.path ?? "") || file.lineSnippet ? "\u200E" : ""}`}>
```